client/src/components/tests/App.test.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import {mount, shallow} from 'enzyme';
 
-import App from './App';
+import App from '../App';
```

Comment:
Looks like you moved your test files to a test directory. Any reason for this? It seems more organized.

Reply:
Yeah I noticed you mentioned that in my last PR! I had been following a testing guide that recommended keeping test files in the same folder as the components they're testing, mostly for easier pathing. But it felt disorganized, and the deliverable for test files ask for a 'link to test folder in github', so I decided to switch to this.